ops/move_deps.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+
+import os
+import re
+import subprocess
+import shutil
+from pathlib import Path
+
+MISSING_REGEX = re.compile(r'\n\t(.+)\ =>\ not\ found')
+FOUND_REGEX = re.compile(r'\n\t(.+)\ =>\ (.+)\ (\(0[xX][0-9a-fA-F]+\))')
+
+def ldd(path):
+    """Get output of ldd for given file"""
+    ldd_out = subprocess.run(
+        ['ldd', path], check=True, capture_output=True, text=True
+    )
+    return ldd_out.stdout
+
+
+def get_missing_deps(ldd_output):
+    """Return iterator of missing dependencies in ldd output"""
+    for match in MISSING_REGEX.finditer(ldd_output):
+        yield match.group(1)
+
+def path_contains(parent, child):
+    """Check if first path contains the child path"""
+    parent = os.path.abspath(parent)
+    child = os.path.abspath(child)
+    return os.path.commonpath([parent]) == os.path.commonpath([parent, child])
```

Comment:
why do you need `os.path.commonpath([parent])` as opposed to just `parent`?

Reply:
I was taught that pattern as a way of ensuring that the same standardizations were applied to both paths being compared, but now that I look at it in context, `abspath` should already standardize the paths sufficiently. I'll go ahead and pull it.